config/initializers/rescue_from.rb

```diff
@@ -0,0 +1,22 @@
+require 'openstax_rescue_from'
+
+OpenStax::RescueFrom.configure do |config|
+  config.raise_exceptions = Rails.application.config.consider_all_requests_local
+
+  config.app_name = 'Accounts'
+  config.app_env = ENV['APP_ENV']
+  config.contact_name = DEPLOY_SETTINGS[:exceptions]['contact_name']
```

Comment:
Well the example file has nesting in it and also I debugged and found that the first level is symbol keys while the second layer is string keys :)